src/server/app.js

```diff
@@ -112,6 +112,14 @@ app.use((req, res, next) => {
 	res.locals.repositoryUrl = repositoryUrl;
 	res.locals.alerts = [];
 
+	if (!req.session) {
+		res.locals.alerts.push({
+			level: 'danger',
+			message: 'We are currently experiencing technical difficulties; ' +
+				'logins will not work until this is resolved.'
```

Comment:
Maybe "signing in will not work until this is resolved." makes more sense?

Reply:
Will address.